freqtrade/freqtradebot.py

```diff
@@ -107,10 +107,7 @@ def worker(self, old_state: None) -> State:
                 Constants.PROCESS_THROTTLE_SECS
             )
 
-            nb_assets = self.config.get(
-                'dynamic_whitelist',
-                Constants.DYNAMIC_WHITELIST
-            )
+            nb_assets = self.config.get('dynamic_whitelist', None)
```

Comment:
nb_assets defaults to `None` instead of `Constants.DYNAMIC_WHITELIST`. Is this intentional?

Reply:
Yes, It's intentional https://github.com/gcarq/freqtrade/pull/603/commits/bb7b2cdfd5b7e3e5c29a425dffbaeb5cecbf09c7